wilma-mock/src/main/java/com/epam/wilma/mock/client/WilmaMock.java

```diff
@@ -289,11 +288,71 @@ public boolean uploadStubConfiguration(String fileName, File file) {
         return fileUpload.uploadStubConfiguration(fileName, file);
     }
 
-    private WilmaMockConfig initializeMockConfig(Properties properties) {
+    /**
+     * For unit test.
```

Comment:
This is not a meaningful doc. Afaik no need for javadoc in case of getter setter, but if it is necessary, pls add "Getter" "Setter" text instead of "For unit test" (applicable for other places in this file too)